omod/src/main/java/org/openmrs/module/web/controller/ModuleListController.java

```diff
@@ -184,18 +184,18 @@ protected ModelAndView onSubmit(HttpServletRequest request, HttpServletResponse
 								ModuleFactory.startModule(depMod);
 								boolean thisModuleCausesRefresh = WebModuleUtil.startModule(depMod, getServletContext(), true);
 								someModuleNeedsARefresh = someModuleNeedsARefresh || thisModuleCausesRefresh;
-								if (someModuleNeedsARefresh) {
```

Comment:
This section of loading servlets and filters was called for each depMod in the for loop. Isn't removing it the same as losing functionality? 😊